tm2/pkg/std/memfile_test.go

```diff
@@ -190,7 +190,16 @@ func TestRePkgOrRlmPath(t *testing.T) {
 			expected: false,
 		},
 	}
-
+	testLengthTable := []struct{
```

Comment:
why dont you add this test case into testTable ?

Reply:
nitpicks : i think we just switch case by the inputs :)

Reply:
I want to separate which cases related to pkg_path's Length, and add more test in future with regex or other methods.

Reply:
@thinhnx-var 
What do you think about doing the following, since this initial test was not valid in the first place:

- Refactor this test to construct a new `MemPackage`, and call `Validate` inside the test run method
- Check the error, instead of the output of a specific unexported function

This way, you can:
- actually validate a public API of `MemPackage`
- utilize the same test cases for both the package name, and package length

I hope you understand the suggestion, I honestly think it's a better approach than defining 2 different sets of test cases, and run methods in the same Go test 🙏

Reply:
@zivkovicmilos 
Thank you very much for your suggestions.
I will: 
- remove the newly added set of testcase 
- add the length limit tests to the `TestMemPackage_Validate` testset which will contructs new `MemPackage` and Validate them.

Let me know if my testing approach is unlike your idea 🙏

Reply:
I will mark this as resolved

Reply:
@thinhnx-var 

This approach sounds good, but you still haven't resolved this in-code

I'm keeping this convo open until you push the revised tests 🙏

Reply:
@zivkovicmilos 
I have just commited. Please review it when you can.
Thank youu.

Reply:
@zivkovicmilos 
Can you take a look on this, and let me know if the refactor code is fine or not.

Reply:
@zivkovicmilos 
I have just updated the test file as Manfred's review
Let me know if you still feel bad of this 🙏